File: beginner/__2000__/2143/2143.py
t = int(input())
while t != 0:
    for _ in range(t):
        n = int(input())
        if (n - 1) % 2 == 0:
            print(2 * n - 1)
        else:
            print(2 * n - 2)
    t = int(input())

File: beginner/__2000__/2143/2143.ts
const fs = require("fs")
const input = fs.readFileSync("/dev/stdin", "utf8").split("\n").map(Number)

let t = input.shift()
while(t != 0){
    for(let i = 0; i < t; i++){
        const n = input.shift()
        console.log((n - 1) % 2 == 0 ? 2 * n - 1: 2 * n - 2 )
    }
    t = input.shift()
}